plugins/module_utils/ansible_freeipa_module.py

```diff
@@ -108,7 +108,9 @@ def parse(version_str):
     try:
         from collections.abc import Mapping  # noqa
     except ImportError:
+        #pylint: disable=wrong-import-position
```

Comment:
As `wrong-import-position` is a regular issue also with plugins, can't it be disabled globally? This is what we currently do in `ansible-freeipa`.